corehq/sql_accessors/sql_templates/get_location_fixture_ids.sql

```diff
@@ -15,43 +15,41 @@ BEGIN
     WITH RECURSIVE expand_to AS (
         WITH RECURSIVE cte AS (
             SELECT
-                "locations_locationtype"."parent_type_id",
+                expand_to_type."parent_type_id",
                 0 AS "depth",
-                "locations_locationtype"."id" AS "expand_to_type"
-            FROM "locations_locationtype"
-            WHERE "locations_locationtype"."id" IN (
-                SELECT U1."expand_to_id"
-                FROM "locations_sqllocation" U0
-                INNER JOIN "locations_locationtype" U1 ON U0."location_type_id" = U1."id"
-                WHERE
-                    U0."id" = ANY(user_location_ids_array)
-                    AND U1."expand_to_id" IS NOT NULL
-            )
+                expand_to_type."id" AS "expand_to_type"
+            FROM "locations_sqllocation" loc
+            INNER JOIN "locations_locationtype" loc_type
+                ON loc."location_type_id" = loc_type."id"
+            INNER JOIN "locations_locationtype" expand_to_type
+                ON expand_to_type."id" = loc_type."expand_to_id"
+            WHERE
+                loc."id" = ANY(user_location_ids_array)
+                AND loc_type."expand_to_id" IS NOT NULL
```

Comment:
now that this is using a join to filter it may return duplicated results. would that be a problem?

Reply:
(this same question applies to all the places in this commit where this was done)

Reply:
Shouldn't be a problem here because duplicates in this query are collapsed by the `SELECT ... FROM "cte" ... GROUP BY ...` below.

I think it also does not matter in the other places where this was done because duplicates in `expand_from` are discarded by `SELECT DISTINCT`, and the final one is in a subquery.